indico/modules/rb/schemas.py

```diff
@@ -352,6 +358,7 @@ class CreateBookingSchema(mm.Schema):
     room_id = fields.Int(required=True)
     booked_for_user = Principal(data_key='user', allow_external_users=True)
     booking_reason = fields.String(data_key='reason', validate=validate.Length(min=3), required=True)
+    notes = fields.String(data_key='notes', validate=validate.Length(min=3), required=False)
```

Comment:
is the `data_key` needed?

Reply:
Nope, I just was following what the `booking_reason` was doing when I first implementing the `notes` system in general, as its pretty similar.